src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -197,8 +197,21 @@ bool ResetGlobalPropertiesRequest::ResetHelpPromt(
     SendResponse(false, mobile_apis::Result::APPLICATION_NOT_REGISTERED);
     return false;
   }
+  const std::vector<std::string>& help_prompt =
+      application_manager_.get_settings().help_prompt();
+
   smart_objects::SmartObject so_help_prompt =
       smart_objects::SmartObject(smart_objects::SmartType_Array);
+
+  for (uint32_t i = 0; i < help_prompt.size(); ++i) {
```

Comment:
@ValeriiMalkov What if size will return 64bit value?

Reply:
@AByzhynar That's really not correct. I changed it to `size_t` type which returns method 'size()' from 'std::vector' container. See commit: 0f55c360d24a552e95417775277ee1d62c39ed43